deploy/operator.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         type: Recreate
       containers:
         - name: kubevirt-ssp-operator
-          image: REPLACE_IMAGE
+          image: quay.io/shwetaap/kubevirt-ssp-operator-container:latest
```

Comment:
keep the `REPLACE_IMAGE` here. It is used by HCO to find the correct place to replace the image